tests/bsim/bluetooth/ll/multiple_id/prj.conf

```diff
@@ -12,19 +12,20 @@ CONFIG_BT_AUTO_DATA_LEN_UPDATE=y
 
 # Maximum simultaneous ACL connections, current maximum considering other roles
 # using ticker nodes count up to 255 (UINT8_MAX) in the hci_ipc with ISO support
-CONFIG_BT_MAX_CONN=231
-CONFIG_BT_ID_MAX=231
+CONFIG_BT_MAX_CONN=230
```

Comment:
Should the multiple_id test change be in a separate commit? It is also not clear why it is changed.

Reply:
No, keeping it in the same commit as `CONFIG_BT_CTLR_ADV_ISO_SET=2` is an increase by one, hence reducing the number of supported simultaneous connections by one.